plugin/evm/orderbook/contract_events_processor.go

```diff
@@ -704,17 +705,30 @@ func (cep *ContractEventsProcessor) updateMetrics(logs []*types.Log) {
 
 		metricName := fmt.Sprintf("%s/%s", "events", event_.Name)
 
-		if !event.Removed {
-			metrics.GetOrRegisterCounter(metricName, nil).Inc(1)
-		} else {
-			metrics.GetOrRegisterCounter(metricName, nil).Dec(1)
-		}
```

Comment:
why did you delet this code? Don't we want to decrement on reorgs?

Reply:
i deleted cuz `updateMetrics` is called only at BlockAccepted, so removed is always false